lib/phoenix-campaign.js

```diff
@@ -13,6 +13,9 @@ class PhoenixCampaign {
     this.status = data.status;
     this.uri = data.uri;
     this.type = data.type;
+    this.currentRun = {
+      en: data.campaign_runs.current.en.id,
```

Comment:
This is [what gets returned from the API:](https://www.dosomething.org/api/v1/campaigns/2070)
````
{

campaign_runs: {
  current: {
    en: {
      id: "6270"
    }
  },

````